calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,169 @@
+
+The yaw-only rotation method, on the other hand, utilizes the initial radar-to-lidar transformation to calculate lidar detections in the radar frame. We then calculate the average yaw angle difference of all pairs, considering only yaw rotation between the lidar and radar detections in the radar frame, to estimate a yaw-only rotation transformation in the radar frame. Finally, we estimate the transformation between the lidar and radar by multiplying the yaw-only rotation transformation with the initial radar-to-lidar transformation.
+
+Generally, the 2d SVD-based method is preferred when valid; otherwise, the yaw-only rotation method is used as the calibration output.
+
+### Diagram
+
+Below, you can see how the algorithm is implemented in the `marker_radar_lidar_calibrator` package.
+
+![marker_radar_lidar_calibrator](../docs/images/marker_radar_lidar_calibrator/marker_radar_lidar_calibrator.jpg)
+
+## ROS Interfaces
+
+### Input
+
+| Name                     | Type                            | Description                                             |
+| ------------------------ | ------------------------------- | ------------------------------------------------------- |
+| `input_lidar_pointcloud` | `sensor_msgs::msg::PointCloud2` | The topic of the lidar pointcloud used for calibration. |
+| `input_radar_msg`        | `radar_msgs::msg::RadarTracks`  | The topic of the radar objects used for calibration.    |
+
+### Output
+
+| Name                          | Type                                   | Description                                     |
+| ----------------------------- | -------------------------------------- | ----------------------------------------------- |
+| `lidar_background_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Background pointcloud from the lidar.           |
+| `lidar_foreground_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Foreground pointcloud from the lidar.           |
+| `lidar_colored_clusters`      | `sensor_msgs::msg::PointCloud2`        | Colored pointcloud clusters from the lidar.     |
+| `lidar_detection_markers`     | `visualization_msgs::msg::MarkerArray` | Lidar detections.                               |
+| `radar_background_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Background pointcloud from the radar.           |
+| `radar_foreground_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Foreground pointcloud from the radar.           |
+| `radar_detection_markers`     | `visualization_msgs::msg::MarkerArray` | Radar detections.                               |
+| `matches_markers`             | `visualization_msgs::msg::MarkerArray` | Matched lidar and radar detection.              |
```

Comment:
detections

I would add it as a suggestion, but it may destroy the layout

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/f3d6bda2eb408d1553e3fb45cb77c35544437e22